src/lib/tutorial.js

```diff
@@ -2,10 +2,6 @@ import { introJs } from 'intro.js'
 import { shouldEnableTracking, getTracker } from 'cozy-ui/react/helpers/tracker'
 require('../../node_modules/intro.js/minified/introjs.min.css')
 
-export function isTutorial () {
-  return window.location.hash.endsWith('/?intro')
```

Comment:
I think we should keep the logic about testing if we are at the tutorial step here, for two reasons:
* The way we determine it is a little dirty and I feel better if it is just encapsulated somewhere
* The condition is not enough self-explanotory. It seems ok when we look at commit, but having just a test about `?intro` in the location does not give enough functional context. Keeping the function and its name here tells us (or a future new developer) that we are at the tutorial step, even without having to look at the code in the conditonal block.

What do you think ?

Reply:
"The way we determine it is a little dirty"
> Yep.
"The condition is not enough self-explanotory."
> Yep too.

I just restore the isTutorial, that's stay a little dirty, but it's enough to sold the bug without making some refactor.